### PR TITLE
upload one file under node.js

### PR DESCRIPTION
**Pull request by KutsenkoA**
Provide FormData to the FileUploader class (FormData is different for node/browser), create multipart/form-data request and upload one file to the fileset
https://jexia.myjetbrains.com/youtrack/issue/JEXIA-1602

### CODE REVIEW

File: e2e/management.ts [hunk 1]
```diff
@@ -99,9 +138,40 @@ export class Management {
...
 
   public deletePolicy(id: string): Promise<any> {
-    return fetch(api.policy.delete.replace("{policy_id}", id), {
+    return this.fetch(api.policy.delete.replace("{policy_id}", id), {
+      method: "DELETE",
+      headers: this.headers
+    });
+  }
+
+  public createFileset(name: string): Promise<any> {
```
[the remark this answers is not in the document]
Comment (KutsenkoA) [reply e2e/management.ts]: In this case, we will end up describing all possible object interfaces here, I didn't want that. It's just a helper methods for creating a test environment. Forgive me this small laziness :)

File: src/api/fileops/fileUploader.ts [hunk 2]
```diff
@@ -28,51 +37,50 @@ export class FileUploader<T, F> {
    * @param file {FilesetMultipart<T, F>}
    */
   private uploadFile(file: FilesetMultipart<T, F>): Observable<IFileUploadStatus> {
+
+    // TODO Append field values here
+    this.formData.append('data', '{}');
+
+    this.formData.append('file', file.file);
```
[the remark this answers is not in the document]
Comment (KutsenkoA) [reply src/api/fileops/fileUploader.ts]: sure, fixed

File: e2e/management.ts [hunk 3]
```diff
@@ -24,14 +62,15 @@ export class Management {
...
+    return this.fetch(api.login, {
       method: "POST",
       headers: {
         "Content-Type": "application/json"
       },
       body: JSON.stringify({
         email: process.env.E2E_EMAIL,
-        password: process.env.E2E_PASSWORD
+        password: process.env.E2E_PASSWORD,
+        recaptchaToken: RECAPTCHA_TOKEN,
```
[the remark this answers is not in the document]
Comment (KutsenkoA) [reply e2e/management.ts]: Did it.

File: e2e/management.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (marlosin) [reply e2e/management.ts]: oh, understood. totally accepted :rofl:

**Pull request merged**